### ISSUE TITLE
Clunky initialization when using multiple views

### ISSUE THREAD

**Issue by alecjacobson** [libigl/libigl#1301]
#### Describe your issue

Is it necessary to append multiple cores during `callback_init`, like the example https://github.com/libigl/libigl/blob/master/tutorial/108_MultipleViews/main.cpp#L18 ?

If so, one would have to replace the simple `.launch()` call with a `.launch_init()`, etc. just to set the background colors (or other properties of each core).

Looking into `viewer.init()` and `core.init()`, these functions don't seem to really do anything and don't rely on OpenGL being initialized.

Unless there's a good reason, I'd like to change the https://github.com/libigl/libigl/blob/master/tutorial/108_MultipleViews/main.cpp example to simply append cores during the normal execution of the `main` function.

#### Check all that apply (change to `[x]`)
- [x] Windows
- [x] Mac OS X
- [x] Linux
- [x] I have tried the `dev` branch and the problem persists

See https://libigl.github.io/CONTRIBUTING/#bugreport for more tips.

**Comment by alecjacobson** [comment]
It appears there is a weird order dependency. `viewer.launch_init()` sets the viewport of the _first_ core to be aligned with the _first_  data (mesh). I recommend we change this to loop over all cores and set the default view for all.

**Comment by jdumas** [comment]
I agree this setup is a bit clunky. I don't see why it wouldn't work the way you suggest. The only kinda weird part about the `launch()` is when you have multiple viewport and you're starting either fullscreen or with a specified window size, then you need to take care to set each `core.viewport` to the correct value.

The `core().align_camera_center(data().V,data().F);` in the `launch_init()` is also probably a remnant from when there was only one core, and can be updated too.
